getting_started/first_3d_game/09.adding_animations.rst

```diff
@@ -271,7 +271,8 @@ following line.
 
    func initialize(start_position, player_position):
        #...
-       $AnimationPlayer.speed_scale = random_speed / min_speed
+       
+       $AnimationPlayer.speed_scale = float(random_speed) / min_speed
```

Comment:
Actually these values should be floats. They are only used in a float context and it is perfectly sensible for a mob to have a speed of `10.5` meters per second. https://github.com/godotengine/godot-docs/pull/6243#discussion_r1133403500

Reply:
right, I think we can change `var random_speed = randf_range(min_speed, max_speed)` instead of `randi-range`. But I also like the to keep it an int and teach the newbies about integer/float division.

Reply:
I'm sure we can find some other place to teach this concept. It doesn't seem right to purposefully keep this code wrong (aka using ints) for the sake of showing how to work around a problem.

Reply:
Yep that's better, I agree